src/main/java/strategies/SearchStrategy.java

```diff
@@ -0,0 +1,10 @@
+package strategies;
```

Comment:
`impl` package should be a subpackage of `strategies` one.

Reply:
fixed